examples/end-to-end-session-based/03-End-to-end-session-based-with-Yoochoose-TF.ipynb

```diff
@@ -0,0 +1,1015 @@
+{
```

Comment:
<p>Maybe we could add one line explaining why we need the <code>.map(lambda X, y: (X, []))</code> (i.e. we set the targets to [] in the data-loader as the true item labels are computed internally by the <code>MaskSequence</code> class ) ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/341/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
sure.